src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -114,20 +115,21 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 
 		}
 
+		HttpClient httpClient = new DefaultHttpClient();
```

Comment:
For try with resources I think we would need to switch to CloseableHttpClient (from the HttpClient) but unfortunately the CloseableHttpClient is not supported by Android. So I left the HttpClient in place for now.  Can modify this when we decide how to proceed with wrapper migration.